README.md

```diff
@@ -58,6 +58,7 @@ In the following table you can find what are supported Kubernetes versions for e
 
 | KubeOne version | 1.14 | 1.13 | 1.12 | Supported providers |
 |-----------------|------|------|------|---------------------|
+| v0.6.0 | + | + | - | AWS, DigitalOcean, GCE, Hetzner, Packet, OpenStack |
```

Comment:
Please also add Packet to the Features section.

Reply:
done